presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduSessionProperties.java

```diff
@@ -38,6 +42,11 @@ public KuduSessionProperties(KuduClientConfig kuduConfig)
                 "Enable grouped execution.",
                 kuduConfig.isGroupedExecutionEnabled(),
                 false))
+            .add(durationProperty(
+                DYNAMIC_FILTERING_WAIT_TIMEOUT,
+                "Duration to wait for completion of dynamic filters",
+                kuduConfig.getDynamicFilteringWaitTimeout(),
```

Comment:
please validate that provided timeout >= 0

Reply:
@MinDuration("0ms") on the getter should do the validation

Reply:
added validation in decoder

Reply:
@sopel39 do we really need special handling for negative values ? I see that `Duration.valueof` will either fail in the pattern match step or a negative value check in the constructor of Duration.
E.g.:
```
set session hive.dynamic_filtering_probe_blocking_timeout='-1s';
Query 20201101_182134_00005_ydqyw failed: duration is not a valid data duration string: -1s
```